### ISSUE TITLE
[Bug]: Highlight Current Page Button on Navigation in Home section🔄

### ISSUE THREAD

**Issue by LitZeus** [mansiruhil13/Bobble-AI#222]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Currently, when navigating from the home page to another page, the button of the tapped navigation element gets highlighted instead of the button corresponding to the Home page (When opening in new tab/window). This issue occurs only on the home page. The expected behavior is to keep the button for the current page highlighted to provide clear visual feedback about the active section. 🖱️✨

### Add Screenshots

https://github.com/user-attachments/assets/0a1009c9-7a23-4ca1-b50d-855409fe3263

### What browsers are you seeing the problem on?

Microsoft Edge

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC contributor
- [X] I want to work on this issue
- [X] I'm willing to provide further clarification or assistance if needed.

**Comment by mansiruhil13** [comment]
yes but its' not the issue. issue now is: when click on home error occured.
**screenshot**
![Screenshot 2024-10-06 200844](https://github.com/user-attachments/assets/33c30fb8-a109-4092-94ca-03f18fb777d2)
you can fix this

**Comment by mansiruhil13** [comment]
its' resolved now
